waku/waku.go

```diff
@@ -1083,7 +1083,7 @@ func (w *Waku) HandlePeer(peer common.Peer, rw p2p.MsgReadWriter) error {
 	w.peers[peer] = struct{}{}
 	w.peerMu.Unlock()
 
-	w.logger.Debug("handling peer", zap.String("id", types.EncodeHex(peer.ID())))
+	w.logger.Info("handling peer", zap.String("id", types.EncodeHex(peer.ID())))
```

Comment:
Wrong field name in log. It's `id` and should be `peerID` as per Logstash parsing rules:
https://github.com/status-im/infra-hq/blob/0a91d7eef311d3e25f25ff3098a7d6964db60013/ansible/roles/logstash/templates/pipe/50-filter-statusd.conf#L14-L20

Reply:
done!